sound/soc/sof/intel/hda.c

```diff
@@ -720,6 +721,19 @@ void hda_dsp_dump(struct snd_sof_dev *sdev, u32 flags)
 	}
 }
 
+void hda_ipc4_dsp_dump(struct snd_sof_dev *sdev, u32 flags)
```

Comment:
If there will be a need for another round of update then I would be inclined to update this commit title and message to clarify that this is the IPC4 fw panic support, we are dealing with the panic in case of IPC3...
Also I would note that this is only available on CAVS2.5+

Reply:
Thanks, updated for ipc4.